README.rst

```diff
@@ -1,7 +1,9 @@
-##############
+==============
 analysis_tools
-##############
+==============
 
-``analysis_tools`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+The Analysis Tools package is designed to assist in the creation of quality assurance (QA) plots and metrics from the outputs of a data reduction pipeline.
```

Comment:
```suggestion
The Analysis Tools package is designed to assist in the creation of quality assurance (QA) plots and metrics from the outputs of a PipelineTask.
```

Reply:
"data reduction pipeline" is either too specific (not just DRP!) or not specific enough (it has to be one of our LSST Pipelines). Maybe "from the outputs of an LSST Pipeline" and link the latter two to `https://pipelines.lsst.io/modules/lsst.pipe.base/creating-a-pipeline.html`? I don't know if we have a better summary page for what we mean when we say "Pipeline", but it's a more specific thing than what non-LSST people mean.

Reply:
DRP != Data Reduction Pipeline.
DRP = Data Release Production. 

Does that change your thinking? I think the phrase "data reduction pipeline" is sufficiently instantly recognisable as to what we mean here - namely, a pipeline that has reduced data. 

Happy to add a URL link.

Reply:
The link helps. We are specifically talking about our capital-P Pipelines, and that makes it clear.